common/src/main/java/org/opensearch/ml/common/transport/model/MLModelGetRequest.java

```diff
@@ -30,21 +30,25 @@
 public class MLModelGetRequest extends ActionRequest {
 
     String modelId;
+    boolean returnContent;
 
     @Builder
-    public MLModelGetRequest(String modelId) {
+    public MLModelGetRequest(String modelId, boolean returnContent) {
```

Comment:
[nit] May be we can rename it to: `isReturnModelContent`

1. returnContent doesn't quite represent model content
2. https://www.samanthaming.com/tidbits/34-better-boolean-variable-names/